src/main/java/staffconnect/logic/commands/DeleteMeetingCommand.java

```diff
@@ -0,0 +1,94 @@
+package staffconnect.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static staffconnect.logic.parser.CliSyntax.PREFIX_MEETING_INDEX;
+
+import java.util.List;
+
+import staffconnect.commons.core.index.Index;
+import staffconnect.commons.util.ToStringBuilder;
+import staffconnect.logic.Messages;
+import staffconnect.logic.commands.exceptions.CommandException;
+import staffconnect.model.Model;
+import staffconnect.model.meeting.Meeting;
+import staffconnect.model.person.Person;
+
+/**
+ * Deletes a meeting identified using it's displayed index from the person's meeting list.
+ */
+public class DeleteMeetingCommand extends Command {
+
+    public static final String COMMAND_WORD = "meeting-delete";
```

Comment:
```delete meeting``` would not work because ```delete``` command already exist and the preamble would be meeting in this case which would throw an error instead, hence the clash with the command ```delete```. The reason why I chose ```meeting-delete``` was to make sure that all the relevant, was so that all command that extended the features of meeting would be appended to the back of meeting with ```-``` such that we get ```meeting-[featureName]```

Reply:
oh thanks for the explanation! the clash in command words slipped my mind.